### ISSUE TITLE
NC14 final: Password confirmation

### ISSUE THREAD

**Issue by jookk** [nextcloud/server#11148]
### Steps to reproduce
1. do something what needs to be password confirmed (remove app,)
2.
3.

### Expected behaviour
I can confortably enter my password to confirm changes.

### Actual behaviour
Password request windows is shown for too short period of time. I have no time to enter my pass, so now window dismiss automaticaly and I cant do anything. 

### Server configuration

**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**14.0.0.19:** (see Nextcloud admin page)

**Updated from an older Nextcloud/ownCloud or fresh install:**

**Where did you install Nextcloud from:**

**Signing status:**
<details>
<summary>Signing status</summary>

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or 

Insert your config.php content here. 
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** yes/no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

#### LDAP configuration (delete this part if not used)
<details>
<summary>LDAP config</summary>

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your Nextcloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```
</details>

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
<details>
<summary>Web server error log</summary>

```
Insert your webserver log here
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
Insert your Nextcloud log here
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```
</details>

**Comment by blizzz** [comment]
@skjnldsv were there reasons for the choice of 7s? sounds indeed too quick. Especially, if you perhaps first need to copy from your password manager.

**Comment by blizzz** [comment]
Otherwise, it's a low hanging fruit for an open source contribution https://github.com/nextcloud/server/blob/master/settings/src/store/api.js#L89 :)

**Comment by skjnldsv** [comment]
Duplicate of #11224

**Comment by mctehlers** [comment]
Is there any plan to solve this? This is seriously annoying when having to copy long passwords from password managers, and when managing users.

**Comment by kesselb** [comment]
Fixed with Nextcloud 14.0.4. Check https://github.com/nextcloud/server/issues/11224 for more details.
